Library/Formula/doxygen.rb

```diff
@@ -19,56 +19,25 @@ class Doxygen < Formula
 
   deprecated_option "with-dot" => "with-graphviz"
 
+  depends_on "cmake" => :build
   depends_on "graphviz" => :optional
   depends_on "qt" if build.with? "doxywizard"
   depends_on "llvm" => "with-clang" if build.with? "libclang"
 
-  # Fixes for --with-doxywizard and --with-libclang
-  patch :DATA
-
   def install
-    args = ["--prefix", prefix]
-
-    if build.with? "doxywizard"
-      args << "--with-doxywizard"
-      ENV["QTDIR"] = Formula["qt"].opt_prefix
-    end
-
-    if build.with? "libclang"
-      args << "--with-libclang-static"
-      llvm = Formula["llvm"]
-      inreplace "configure" do |s|
-        s.gsub! /libclang_hdr_dir=\".*$/, "libclang_hdr_dir=\"#{llvm.opt_include}\""
-        s.gsub! /libclang_lib_dir=\".*$/, "libclang_lib_dir=\"#{llvm.opt_lib}\""
-      end
-    end
-
-    system "./configure", *args
-
-    # Per MacPorts:
-    # https://trac.macports.org/browser/trunk/dports/textproc/doxygen/Portfile#L92
-    inreplace %w[ tmake/lib/macosx-c++/tmake.conf
-                  tmake/lib/macosx-intel-c++/tmake.conf
-                  tmake/lib/macosx-uni-c++/tmake.conf ] do |s|
-      # makefiles hardcode both cc and c++
-      s.gsub! /cc$/, ENV.cc
-      s.gsub! /c\+\+$/, ENV.cxx
-    end
-
-    # This is a terrible hack; configure finds lex/yacc OK but
-    # one Makefile doesn't get generated with these, so pull
-    # them out of a known good file and cram them into the other.
-    lex = ""
-    yacc = ""
-
-    inreplace "src/libdoxycfg.t" do |s|
-      lex = s.get_make_var "LEX"
-      yacc = s.get_make_var "YACC"
+    args = std_cmake_args + %W[
+      -DCMAKE_INSTALL_PREFIX=#{prefix}
```

Comment:
@mikemcquaid Good catch
@DomT4 Thanks for the info.

I've updated the PR accordingly and also remove the `-Dbuild_doc=ON` option, since that isn't necessary with the manual installation in this formula (it was a left-over from previous experiments).